lib/byron/bench/Restore.hs

```diff
@@ -190,16 +191,16 @@ main = do
     let args =
             [ "run"
             , "--database-path", nodeDB
-            , "--topology", configs </> "mainnet-topology.json"
+            , "--topology", topology
             , "--socket-path", socketPath
-            , "--config", configs </> "configuration-mainnet.yaml"
+            , "--config", config
             , "--port", "7776"
             ]
-    -- NOTE: It is warned against using @NoStream@ to supress output on POSIX
-    -- systems, since it could lead to the process throwing errors.
-    -- It appears to work fine stopping the live view of cardano-node,
-    -- however.
-    let cmd = Command "cardano-node" args (return ()) NoStream NoStream
+    let cmd = Command "cardano-node" args (return ()) Inherit Inherit
```

Comment:
The node config is not using `LiveView` anymore but `SimpleView`, which indeed breaks when we use `NoStream`. Using `silently` seemed complicated since we need to change `withBackendProcess` or `Command`. I found the current output to stdout to be fine though, and shouldn't contaminate any data.

Reply:
https://buildkite.com/input-output-hk/cardano-wallet/builds/6731#9c9e2709-e4a2-4574-8d06-b0aef9221c68 Logs exceed 1MB… should probably limit them.

Reply:
https://github.com/input-output-hk/cardano-wallet/pull/1521/commits/c48cbee1cd7cf488e4df0ee684655f9c107588d8

Reply:
Maybe not in this PR, but it would be good if cardano-node logs were still written to file (perhaps even uploaded as buildkite artifacts). Then we can refer back to them if a benchmark runs badly.